Backpack/Classes/Font/Generated/BPKFont.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@class UIFont;
+@interface BPKFont: NSObject
+
++ (UIFont *)textXs;
```

Comment:
Tip: add `/// System font of size 11, regular weight` or similar. I at least tend to forget the size and weights.

Reply:
The use of Backpack at Skyscanner means design docs will already be specifying one of these sizes so when using it you shouldn't have to care about the underlying values.